### ISSUE TITLE
Search commands

### ISSUE THREAD

**Issue by HeiWiper** [TBog/TBLauncher#89]
**Description**
Sometimes the user wants to search for let's say a location, he would have to search for the navigation app and then launch it to finally search for his desired location. 

**Solution**
I would like to suggest adding search commands e.g user sets `y` to be the shortcut for Youtube then he types `y search query` in the TinyBit Search Bar to get results for that search query.
This would automatically open the default app for the URL if the user has allowed the app to open that URL, so it would save a few clicks.

**Additional context**
[Blue Line Console (Keyboard based launcher)](https://f-droid.org/app/net.nhiroki.bluelineconsole) is an app that does the same thing except it's an assistant app not a home launcher.

![Screenshot](/storage/emulated/0/Pictures/Screenshots/Screenshot_20210115-193153184.jpg)

**Comment by HunterArmstrong** [comment]
You could probably just add a tag then look it up if you have  the "Fuzzy-search Tags" setting on in the features section of the settings.

**Comment by HeiWiper** [comment]
> You could probably just add a tag then look it up if you have the "Fuzzy-search Tags" setting on in the features section of the settings.

I think I wasn't clear enough, I just fixed the screenshot it wasn't showing up, can you see it and tell me if that's what you were talking about ?

**Comment by HunterArmstrong** [comment]
So you were talking about an internet search key word. Am I getting this?

**Comment by HeiWiper** [comment]
Yes, I might need to change the title but that's basically what I was referring to.

**Comment by HunterArmstrong** [comment]
Gooootcha
